CHANGELOG.md

```diff
@@ -4,6 +4,97 @@ All notable changes to Zebra are documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-beta.14](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-beta.13) - 2022-08-30
+
+This release contains a variety of CI fixes, test fixes and dependency updates.
+
+### Added
+
+- Store history trees by height in the non-finalized state (#4928)
+- Breaking: Add `start` as default subcommand for `zebrad` (#4957)
+
+### Changed
+
+- Revert deserializing stored transactions in a rayon thread (#4933)
```

Comment:
Motivation for the PR describes [#4933](https://github.com/ZcashFoundation/zebra/pull/4933) as a fix

Reply:
```suggestion
- Fix a performance regression when serving blocks via the Zcash network protocol and RPCs (#4933)
```

Reply:
Thanks for catching this!